### DISCUSSION TITLE
Output file name

### DISCUSSION THREAD

**Discussion by kahaboom** [rfennell/AzurePipelines#1411] [Q&A] [answered]
Hello @rfennell 
I am facing the next issue. Cannot find any info regarding this. 
When the release notes are generating, it is creating a file. 
<img width="806" alt="Screenshot 2022-12-01 at 11 31 38" src="https://user-images.githubusercontent.com/80584072/205017317-489926e0-6d84-4145-acc2-388a5048fef0.png">

what I want to ask, how can I provide a name for the output file so it the same as build pipeline that is triggering this releases ?
For example, here the pipeline build is #11

<img width="476" alt="Screenshot 2022-12-01 at 11 32 21" src="https://user-images.githubusercontent.com/80584072/205017564-aaad76d4-4285-4293-ab4a-228c0195d241.png">

Could you help with it please.
Thank you.

**Comment by rfennell** [comment, marked as answer]
The output filename is set using standard task parameter, so you can build the name using any custom or [predefined variables
](https://learn.microsoft.com/en-us/azure/devops/pipelines/build/variables?view=azure-devops&tabs=yaml)

Maybe something like this

`$(Build.ArtifactStagingDirectory)/$(Build.Buildnumber).md`

Or maybe

`$(Build.ArtifactStagingDirectory)/$(Build.buildid).md`

**Reply by kahaboom** [reply]
Thanks for answer and suggest @rfennell 
I've found out that it can be given just by providing this -` $(Build.Buildnumber).md` . No need to use `$(Build.ArtifactStagingDirectory)` variable.
